scripts/build_vcpkg_deps.ps1

```diff
@@ -124,6 +129,32 @@ function run_script {
         Copy-Item -Path ".\bin\$file" -Destination ".\tools"
     }
     
+    Write-Output "Copying files to build folder"
+
+    # Create the Debug and Release folder if they do not exist (fresh folder)
+    # We expect it to be built to *BuildFolder*\CorsixTH\(Debug|Release)
+    $debug_build_path = $BuildFolderAbsPath + "\CorsixTH\Debug"
+    $release_build_path = $BuildFolderAbsPath + "\CorsixTH\Release"
+
+    # Create the destination folders if they do not exist
+    # Have to pipe to Out-Null as they print the result which adds noise to
+    # the CMake console
+    if (-Not (Test-Path -Path $debug_build_path)){
+        New-Item -ItemType Directory -Force -Path $debug_build_path | Out-Null
+    }
+    if (-Not (Test-Path -Path $release_build_path)){
+        New-Item -ItemType Directory -Force -Path $release_build_path | Out-Null
+    }
+
+    # Copy all tools to the output folders
+    Copy-Item -Force -Path ".\tools\*" -Destination $debug_build_path
```

Comment:
We don't copy `tools` to the output path. The only thing we need `tools` for is LUA_PROGRAM_PATH to run LDocGen. We also need lfs.dll and lpeg.dll in the `tools` directory for that to work (or we copy everything out to another directory like I did in my appveyor PR)

Reply:
So just to clarify we don't need top copy any files from the tools directory out? 
The earlier changes copied `lfs.dll` and `lpeg.dll` into the tools directory and I was unsure if they needed copying into the build directory.

Reply:
`lfs.dll` and `lpeg.dll` need to be in the output directory; they should still be in the `bin` folder though. The tools folder only contains `lua.exe` and `luac.exe`, which we don't need to distribute.